crates/extensions/c8y_mapper_ext/src/tests.rs

```diff
@@ -2104,25 +2102,14 @@ async fn custom_operation_timeout_sigkill() {
         &mut mqtt,
         [(
             "c8y/s/us",
-            "502,c8y_Command,\"operation failed due to timeout: duration=1s\"",
+            "502,c8y_Command,\"operation failed due to timeout: duration=2s\"",
         )],
     )
     .await;
 
-    // assert the signterm is handled
-    let expected_content = "command \"511,test-device,c8y_Command\"
-exit status: unknown
-
-stdout <<EOF
-main 1
-sigterm 1
-sigterm 2
-EOF
-
-stderr <<EOF
-operation failed due to timeout: duration=1sEOF
-";
```

Comment:
Why this has been removed and replaced by a weaker assertion?

Reply:
As, mentioned in the other comment, when the `sigterm` is sent for gracefully complete the task, this might be handled and the handler will start printing the `sigterm x` logs. Because still not gracefully stopped, the operation will be killed forcefully by sending the `sigkill`, which will take some time to stop. In some cases, the handler logs 2 times and in some cases 3 times. This is the reason it will be hard to assert with the fixed set of log messages. So, I introduced a weaker assertion. I feel it is enough to assert whether the `sigterm` handled or not and also whether the `sigkill` stopped the operation with the status `unknown` or not.